### PR TITLE
Results screen

### LINKED ISSUE #22: Score system UI implementation
**Minimum Acceptance Criteria**
Display the amount of points earned in a round, and total points earned per player.

**Is blocked by the following issues**
#21 
#122

### LINKED ISSUE #122: Score system UI design
**Minimum Acceptance Criteria**
Create a design for displaying score at the end of each round in the client

**Is blocked by the following issues**
#21

### PR DESCRIPTION
**Pull request by Zemanzo**
Closes #122 
Closes #22 

Also in this PR: Removed useless overlay buttons in editor; Minor fixes in Terms & Conditions and Privacy pages; Discord badge in README is now Blurple; Badges now have logos

The only thing missing right now is the lack of WR tracking per map, which will be addressed in #87

### CODE REVIEW

File: src/client/client/game.js [hunk 1]
```diff
@@ -139,6 +150,121 @@ let game = (function() {
...
+							return 1;
+						} else if (b.finished) {
+							return -1;
+						} else {
+							return 0;
+						}
+					});
+
+					// Reverse it cause im stupid
+					_enteredMarbleList.reverse();
```
Comment (Qaomen) [inline src/client/client/game.js]: Why not flip the conditions in the sorting function instead? ~~That way you can remove that comment and no one will know what really happened.~~

File: src/server/game.js [hunk 2]
```diff
@@ -45,13 +45,17 @@ let game = function() {
 		_firstMarbleHasFinished = false,
 
 		_gameplayParameters = null,
+		_mapName = null,
```
Comment (Qaomen) [inline src/server/game.js]: Maybe we should have a single `_currentMap` property at this point.

File: src/client/client/game.js [hunk 3]
```diff
@@ -139,6 +150,121 @@ let game = (function() {
...
+					_DOMElements.resultsList.appendChild(resultsListFragment);
+
+					_DOMElements.raceLeaderboardMapName.innerText = newStateData.data.map.name;
+					_DOMElements.raceLeaderboardAuthorName.innerText = newStateData.data.map.author;
+
+					// Automatic scroll
+					if (_DOMElements.resultsList.scrollHeight > 0) {
+						_DOMElements.resultsList.scrollTop = _DOMElements.resultsList.scrollHeight;
+
+						// Start scrolling up
```
Comment (Qaomen) [inline src/client/client/game.js]: It seems more sensible to scroll from top to bottom. Seeing top ranks first is more interesting than the late/DNF'd marbles. (Or maybe allow the user to scroll through?)

**Review by Qaomen — CHANGES_REQUESTED** [review]
In short: This is awesome!
Slightly longer version: This is really awesome! Love the way it fades out and everything too!

The only notable thing is the scroll direction. If/whether it scrolls from top to bottom or gives the user the choice to scroll themselves, I leave up to you.

File: src/client/client/game.js (hunk 1 above)
Comment (Zemanzo) [reply src/client/client/game.js]: ~~cause im stupid~~

File: src/client/client/game.js (hunk 3 above)
Comment (Zemanzo) [reply src/client/client/game.js]: Scrolling will be removed and only the top ranking entries will be shown by default. The user can scroll manually through the list as long as it is on screen. This is done as an incentive to play more and reach a higher rank, to claim a few seconds of fame.

**Review by Qaomen — APPROVED** [review]

**Pull request merged**